client/src/extension.ts

```diff
@@ -60,59 +60,36 @@ export function activate(context: vscode.ExtensionContext) {
 
           // brew install location mac
           // intel
-          "/usr/local/bin/nu",
+          "/usr/local/bin",
           // arm
-          "/opt/homebrew/bin/nu",
+          "/opt/homebrew/bin",
 
           // native package manager install location
-          "/usr/bin/nu",
+          // standard location should be in `PATH` env var
+          //"/usr/bin/nu",
```

Comment:
I'd prefer to leave this folder. Same rationale with "should be" vs "are".

Reply:
same as above, especially this one, because it's a **standard** unix location. if it's not in the `PATH` environment variable, it's almost certainly been configured bymanually, which means we are highly like running in a estoric environment (e.g. containerized), so it's best to just honer the `PATH` variable.

in fact, if we search the location in `env:PATH`, most of those heuristics are redundent anyway. I kept them here just to make the diff clafified.